### PR TITLE
Moved SMS Notifier to CFN template

### PR DESCRIPTION
**Pull request by dstroppa**
Fix for issue #5

### CODE REVIEW

File: sms-notifier/LambdaFunction.js [hunk 1]
```diff
@@ -2,30 +2,28 @@
 var AWS = require('aws-sdk');
 var sns = new AWS.SNS();
```
Comment (pputhran) [inline sms-notifier/LambdaFunction.js]: could possibly use let instead of var

File: sms-notifier/LambdaFunction.js [hunk 2]
```diff
@@ -2,30 +2,28 @@
 var AWS = require('aws-sdk');
```
Comment (dstroppa) [inline sms-notifier/LambdaFunction.js]: pputhran: All the var(s) can be replaced with let as I see we are using ‘const’ if we want to strictly follow ES6

File: sms-notifier/LambdaFunction.js [hunk 3]
```diff
@@ -2,30 +2,28 @@
...
-// define configuration
-const phoneNumber =''; // Insert phone number here. For example, a U.S. phone number in E.164 format would appear as +1XXX5550100
-
 //main function which gets AWS Health data from Cloudwatch event
 exports.handler = (event, context, callback) => {
+    //get phone number from Env Variable
+    var phoneNumber = process.env.PHONE_NUMBER;
     //extract details from Cloudwatch event
     eventName = event.detail.eventTypeCode
     healthMessage = 'The following AWS Health event type has occured: ' + eventName + ' For more details, please see https://phd.aws.amazon.com/phd/home?region=us-east-1#/dashboard/open-issues';
```
Comment (dstroppa) [inline sms-notifier/LambdaFunction.js]: pputhran: The above could be rewritten as
healthMessage = `The following AWS Health event type has occured: - ${eventName}. For more details, please see https://phd.aws.amazon.com/phd/home?region=us-east-1#/dashboard/open-issues`; */

File: sms-notifier/LambdaFunction.js [hunk 4]
```diff
@@ -2,30 +2,28 @@
...
     //prepare message for SNS to publish
     var snsPublishParams = {
-        Message: healthMessage, 
+        Message: healthMessage,
         PhoneNumber: phoneNumber,
     };
     sns.publish(snsPublishParams, function(err, data) {
```
Comment (dstroppa) [inline sms-notifier/LambdaFunction.js]: pputhran: I have also modified the sns.publish function to keep it consistent with ES6 syntax

**Comment by dstroppa** [comment]
Added pputhran comments

File: sms-notifier/LambdaFunction.js (hunk 4 above)
Comment (pputhran) [inline sms-notifier/LambdaFunction.js]: `sns.publish(snsPublishParams,(err,data) => {
         if (err) {
            const snsPublishErrorMessage = `Error publishing AWS Health event to SNS`;
            console.log(snsPublishErrorMessage, err, err.stack); // adding the err.stack
            callback(snsPublishErrorMessage);
        }
 
        const snsPublishSuccessMessage = `Successfully got details from AWS Health event, ${eventName} and sent SMS via SNS.`;
        console.log(snsPublishSuccessMessage, data);
        callback(null, snsPublishSuccessMessage); //return success 
    });`

Comment (pputhran) [inline sms-notifier/LambdaFunction.js]: sns.publish(snsPublishParams,(err,data) => {
         if (err) {
            const snsPublishErrorMessage = `Error publishing AWS Health event to SNS`;
            console.log(snsPublishErrorMessage, err, err.stack); // adding the err.stack
            callback(snsPublishErrorMessage);
        }
 
        const snsPublishSuccessMessage = `Successfully got details from AWS Health event, ${eventName} and sent SMS via SNS.`;
        console.log(snsPublishSuccessMessage, data);
        callback(null, snsPublishSuccessMessage); //return success
 
    });

**Pull request merged**